terms.adoc

```diff
@@ -0,0 +1,130 @@
+////
+Apply pseudo legal capitalization to the main document:
+cat eclipse_development_process.adoc | sed -E -e "`grep -P -oh "^.+(?=\s+\:\:)" terms.adoc | while read -r term; do echo -ne "s|$term|$term|gI;"; done`"
+////
+
+[glossary]
+
+Architecture Council ::
+The Eclipse Architecture Council (AC) is a council of experienced Eclipse committers responsible for identifying and tackling any issues that hinder continued technological success and innovation, widespread adoption, and future growth of Eclipse technology.
+
+Board of Directors ::
+The business and technical affairs of the Eclipse Foundation are managed by or under the direction of the Eclipse Board of Directors. 
+
+Check Point Review ::
```

Comment:
Maybe we should define clarify the goal of a "Check Point Review" here. Something like "A check point review testifies that project is active.". I think instead of "check point", maybe saying "Project Activity Review" is more explicit and easier to grok/use.

Reply:
We don't actually use this term anywhere in the EDP. "Check Point Review" is included here because it is used in the IP Policy.

Reply:
... the implication being that changing this will require a change in the IP Policy which adds to the challenge of timing the roll out (that, and any change to the IP Policy requires review from the IP Advisory Committee and BoD). We should only change this if it's really necessary.

Reply:
That's not really agile... I think it's also important for the AC and EMO at large to sort out ways to better embrace change so we don't get stuck with things that could be improved, just because it's too annoying to change them.
But it's more food for thoughts, not much related to the patch.

Reply:
How about we just remove the term? We've gotten on this far without it defined here and attempting to define it here increases coupling unnecessarily.